openshift-knative-operator/hack/update-manifests.sh

```diff
@@ -35,8 +35,12 @@ function download {
     index=$(( i+1 ))
     file="${files[$i]}.yaml"
     target_file="$target_dir/$index-$file"
-    url="https://github.com/knative/$component/releases/download/$version/$file"
 
+    if [[ "${component}" == "serving" ]]; then
+      url="https://github.com/knative/$component/releases/download/knative-$version/$file"
```

Comment:
can't we do the same `knative-` construction for net-istio and not need the extra download thingy?

Reply:
Do you mean using https://github.com/knative-sandbox/net-istio/releases url and download just the net-istio.yaml instead of iterating over those net-istio files? The reason for this download version thingy is that there is no version to use for the url `"https://raw.githubusercontent.com/knative-sandbox/${component}/${version}/config/${file}"` other than 0.27.0.

Reply:
Sure there is a direct URL to those files, see https://raw.githubusercontent.com/knative-sandbox/net-istio/knative-v1.0.0/config/200-clusterrole.yaml for example. Using the `knative-v1.0.0` tag should be just fine here too.